docs/hugo/content/architecture-decision-records/ADR-2021-08-Committing-Generated-Code.md

```diff
@@ -0,0 +1,59 @@
+# Committing Generated Code
+
+## Context
+
+We’re aware that common practice avoids committing generated code into source control, treating it as an ephemeral resource that can be regenerated on demand.
+
+Even so, we are concluding that we want to check in our generated code.
```

Comment:
minor: Not sure of tense here. Maybe, "we concluded" or "we have concluded"? Since we've made that conclusion and this is documenting how we came to it?

Reply:
The Context section of an ADR is supposed to be written in present tense to describe the context as at that time.